pages/settings/index.tsx

```diff
@@ -1,5 +1,79 @@
+import React from 'react'
+import { MuiChipsInput, MuiChipsInputChip } from 'mui-chips-input'
+import {
+  Box,
+  Checkbox,
+  TextField,
+  Tooltip,
+  Typography,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material'
+import Grid2 from '@mui/material/Unstable_Grid2'
+import { CheckBox } from '@mui/icons-material'
+import InfoIcon from '@mui/icons-material/Info'
+
 export default function SettingsPage() {
+  const theme = useTheme()
+  const [value, setValue] = React.useState<MuiChipsInputChip[]>([])
+  const isMobileView = useMediaQuery(theme.breakpoints.down('md'))
+  const handleChange = (newValue: MuiChipsInputChip[]) => {
+    setValue(newValue)
+  }
+  const emailRegex =
+    /^[a-zA-Z0-9.!#$%&'+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:.[a-zA-Z0-9-]+)$/
```

Comment:
```suggestion
    /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/
```
The regex allows emails with no top-level-domain to be valid, not sure if this is intended. I found this regex and it seems to fix that issue.